src/adapters/ip_camera_adapter/ipcam.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
```

Comment:
So we already have $src_dir/huedemo.sh and now this script. I don't care too much about python/bash/whatever, but they would be easier to find if we put them all in the same place. Maybe in $src_dir/tools/scripts? Or feel free to propose something else.